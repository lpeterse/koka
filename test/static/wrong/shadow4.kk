// combination of warning and error used to display incorrectly
fun foo(x) { x = 2; x + True }