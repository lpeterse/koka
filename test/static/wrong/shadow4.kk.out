test/static/wrong/shadow4.kk(2,14): warning: x shadows an earlier local definition or parameter
test/static/wrong/shadow4.kk(2,25): error: types do not match
 context : x + True
 term : True
 inferred type: bool
 expected type: int